tuned/plugins/plugin_scheduler.py

```diff
@@ -318,15 +318,15 @@ def _tune_process(self, pid, cmd, sched, prio, affinity):
 	def _convert_sched_params(self, str_scheduler, str_priority):
 		scheduler = self._dict_schedcfg2num.get(str_scheduler)
 		if scheduler is None and str_scheduler != "*":
-			log.error("Invalid scheduler: %s. Scheduler and priority will be ignored."
-					% str_scheduler)
+			log.warn("Invalid scheduler: %s. Scheduler and priority will be ignored."
+					 % str_scheduler)
 			return (None, None)
 		else:
 			try:
 				priority = int(str_priority)
 			except ValueError:
-				log.error("Invalid priority: %s. Scheduler and priority will be ignored."
-							% str_priority)
+				log.warn("Invalid priority: %s. Scheduler and priority will be ignored."
```

Comment:
Again, this can only happen if an non-integer priority is specified in the profile, so only due to misconfiguration. I think error level is appropriate here.